ui/apps/platform/src/Containers/Dashboard/Widgets/ViolationsByPolicyCategory.test.tsx

```diff
@@ -5,15 +5,15 @@ import '@testing-library/jest-dom/extend-expect';
 
 import renderWithRouter from 'test-utils/renderWithRouter';
 import { mockChartsWithoutAnimation } from 'test-utils/mocks/@patternfly/react-charts';
-import ViolationsByPolicyCategory from 'Containers/Dashboard/PatternFly/Widgets/ViolationsByPolicyCategory';
+import ViolationsByPolicyCategory from 'Containers/Dashboard/Widgets/ViolationsByPolicyCategory';
```

Comment:
In ViolationsByPolicySeverity.test.tsx below, import path is relative and mock is from Containers level.

What do you think about simpler import path here too?

Reply:
Good catch, I'll push the fix.